app/javascript/packs/recommend_pages.js

```diff
@@ -61,7 +63,7 @@ const CampaignTile = props => (
         })}
       </div>
     </div>
-    <div className="campaign-tile__lead">{props.title}</div>
+    <div className="cggggampaign-tile__lead">{props.title}</div>
```

Comment:
Did you mean to rename this class so wonky?

Reply:
😨 me failing at VIM